test/ng/compileSpec.js

```diff
@@ -9824,37 +9863,6 @@ describe('$compile', function() {
         expect(element.children().eq(2).text()).toEqual('dorothy');
       });
     });
-
-    it('should not overwrite the contents of an `ng-transclude` element, if the matching optional slot is not filled', function() {
```

Comment:
because it is actually an exact duplicate of the previous (or following) test :-) It must have sneaked in there as part of a rebase in the past